icon/iiss/icstate/networkvalue.go

```diff
@@ -118,3 +120,18 @@ func GetTotalStake(s *State) *big.Int {
 func SetTotalStake(s *State, value *big.Int) error {
 	return setValue(s.store, VarTotalStake, value)
 }
+
+func (s *State)GetBondRequirement() int {
+	return int(GetBondRequirement(s))
+}
+
+func GetBondRequirement(s *State) int64 {
+	return getValue(s.store, VarBondRequirement).Int64()
+}
+
+func SetBondRequirement(s *State, value int64) error {
+	if value == 0 || value > 100 {
```

Comment:
Please consider the condition when value is negative.

Reply:
I reflected the case, thank you.